lib/sdkGenHelper.js

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright IBM Corporation 2016-2017
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+var fs = require('fs');
+var chalk = require('chalk');
+var request = require('request');
+var unzip = require('unzip2');
+var stream = require('stream');
+var Rsync = require('rsync');
+var rimraf = require('rimraf');
```

Comment:
rimraf is not in the `dependencies` in the `package.json`. Since it *is* in the `devDependencies` already, it just needs to be moved across. The tests don't fail on this because the module can often be found on newer npm versions due to the way they are installed, but it probably should not be relied upon (and won't work on `npm@2` IIRC).